Mu2eG4/src/Mu2eWorld.cc

```diff
@@ -719,23 +701,37 @@ namespace mu2e {
     }
 
     // and the calorimeter elements
+    // all vacuum volumes in the calorimeter have the same DSVacuum material, so this does them all in one go
+    G4LogicalVolumeStore* lvs = G4LogicalVolumeStore::GetInstance();
+    for (auto lvi = lvs->begin(); lvi != lvs->end(); ++lvi) {
+      if ((*lvi)->GetName().find("Calo")!=std::string::npos &&
+          (*lvi)->GetMaterial()->GetName().find("Vacuum") != std::string::npos) {
+          (*lvi)->SetUserLimits( stepLimit );
+      }
+    }
 
-    stepLimiterHelper("^caloDisk_.*$", stepLimit );
-    stepLimiterHelper("^caloFEB_.*$", stepLimit );
-
-    // calorimeter does not use Mu2e nest functions for those; see above; fixme
-    // stepLimiterHelper("^caloFEBLog*$", stepLimit );
-
-    // stepLimiterHelper("^ccrateFullBoxLog*$", stepLimit );
-    // stepLimiterHelper("^ccrateBoxInLog*$", stepLimit );
-    // stepLimiterHelper("^ccrateBoardLog*$", stepLimit );
+    //vector<G4LogicalVolume*> caloVacua;
```

Comment:
Please remove large chunks of commented out code unless it's being saved for future development or debugging.  Please check all files for this.

Reply:
Yes, that was a huge oversight, I was so happy to have squashed my bug that I forgot to remove the previous code.... I wish I could blame it on beer, but this wasn't even the case!